src/android/CameraLauncher.java

```diff
@@ -101,8 +101,10 @@ Licensed to the Apache Software Foundation (ASF) under one
 
     private static final String LOG_TAG = "CameraLauncher";
 
-    //Where did this come from?
-    private static final int CROP_CAMERA = 100;
+    //Crop Codes
+    private static final int CROP_OPERATION = 100;
+    private static final int CROP_CAMERA = CROP_OPERATION;
+    private static final int CROP_PHOTO_ALBUM = CROP_OPERATION + 900;
```

Comment:
any particular reason for `900`? Can we use something better than a random magic number?

Reply:
The reason is to give a clear distinction between CROP_CAMERA operation code and CROP_PHOTO_ALBUM operation code.

Reply:
Hey @timbru31, why 900 specifically? Is there a reason for it?